test/coverage.js

```diff
@@ -175,6 +175,14 @@ describe('Coverage', () => {
         done();
     });
 
+    it('retains original value of conditional result with comma operator', (done) => {
```

Comment:
You will want to enable coverage on a test that uses the condition2 file to make sure that coverage works and the method executes correctly